canvas_modules/common-canvas/locales/command-actions/locales/en.json

```diff
@@ -16,14 +16,14 @@
 	"action.createNodeOnLink": "Create {node_label} node and insert into link",
 	"action.createSuperNode": "Create {node_label} supernode",
 	"action.deconstructSuperNode": "Deconstruct {node_label} supernode",
-	"action.deleteLink": "Delete link",
-	"action.deleteComments":"Delete {comments_count} comments",
-	"action.deleteNodes":"Delete {nodes_count} nodes",
+	"action.deleteLink": "Delete 1 link",
```

Comment:
You can change this back to 'Delete link' since the action that causes this to be displayed only ever deletes a single link.